tasks/main.yml

```diff
@@ -20,8 +20,7 @@
 
 - name: Download and untar the cyhy-commander tarball
   unarchive:
-    src: "https://api.github.com/repos/jsf9k/cyhy-commander/tarball/develop?\
-    access_token={{ github_oauth_token }}"
+    src: "https://api.github.com/repos/cisagov/cyhy-commander/tarball/develop"
```

Comment:
Is this premature?  Isn't cyhy-commander still a private repo in jsf9k?  Maybe I have confused myself.

Reply:
Yes, you're correct @jsf9k - we should not be making these changes until `cyhy-commander` goes public.

Reply:
:mushroom: :cloud: :boom:

Reply:
Aha! Yes, it is premature. I lost track of commander still being private, so this PR should be on-hold for now. Thanks!

Reply:
No need to hold this one.  I repurposed it to get around the OAuth token deprecation issue.  That's why I changed the title and description.

In retrospect this is pretty tacky on my part, though.  We should create separate PRs instead of repurposing.  I won't do this again.